src/graphql/node.rs

```diff
@@ -461,23 +462,23 @@ impl NodeStatus {
     async fn id(&self) -> ID {
         ID(self.id.to_string())
     }
-
+    /// The RAM size in bytes. The type must be u64.
     async fn total_memory(&self) -> Option<StringNumber<u64>> {
         self.total_memory.map(StringNumber)
     }
-
+    /// The amount of used RAM in bytes. The type must be u64.
     async fn used_memory(&self) -> Option<StringNumber<u64>> {
         self.used_memory.map(StringNumber)
     }
-
+    /// The total disk space in bytes. The type must be u64.
     async fn total_disk_space(&self) -> Option<StringNumber<u64>> {
         self.total_disk_space.map(StringNumber)
     }
-
+    /// The total disk space in bytes that is currently used. The type must be u64.
     async fn used_disk_space(&self) -> Option<StringNumber<u64>> {
         self.used_disk_space.map(StringNumber)
     }
-
+    /// The ping value for a specific node. The type must be u64.
     async fn ping(&self) -> Option<StringNumber<i64>> {
```

Comment:
The types specified in the comment and the return value are not the same.

Reply:
I missed it. 🥲  I replaced `u64` to `i64`.